eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/EnactSpec.hs

```diff
@@ -50,7 +50,6 @@ spec ::
   SpecWith (ImpTestState era)
 spec =
   describe "ENACT" $ do
-    relevantDuringBootstrapSpec
```

Comment:
Instead of removing these, I'd recommend adjusting each affected test and overwrite the threasholds to `0`, whenever `curPV > ProtVer 9`

Reply:
done, thank you